provision-sm-md-lg/README.md

```diff
@@ -0,0 +1,81 @@
+# Provision VM Service Small Medium Large Example
+
+This example shows creating a provisioning dialog that modifies provisioning parameters
+
+## Architecture
+
+- `provision-vm-small-med-large.asl`: This workflow is run for every vm provisioned and is responsible for modifying the vm size. This will be used as a ManageIQ Service's Provisioning Entry Point.
+
+## Setup within ManageIQ
+
+### Set up the workflow repository
+
+#### Add this repository
+
+1. Automation -> Embedded Workflows -> Repositories.
+2. Configuration -> Add new Repository
+3. `Name = Example Workflows`
+4. `URL = https://github.com/ManageIQ/workflows-examples`
+5. Click Save
+6. Verify for a non+zero workflow count.
+
+### Create a Provisioning Dialog
+
+1. Automation -> Embedded Automate -> Customization
+2. Service Dialogs -> All Dialogs
+3. Configuration -> Add a new Dialog
+1. `Name = sm-med-lg dialog`
+1. `Description = Dialog choosing vm size`
+1. Drag a new Text Box to "New section"
+1. `Label = Vm Name`
+1. `Name  = vm_name`
+1. Drag Dropdown to New section
+1. `Label = size`
+1. `Name = size`
+1. Click edit/pencil icon for "size" dropdown
+1. Options -> Entries
+
+   |  Name | Value  |
+   |-------|--------|
+   |Large  | large  |
+   |Medium | medium |
+   |Small  | small  |
+
+1. Click Save
+
+### Create a Service Catalog Item
+
+1. Service > Catalogs
+2. Catalogs > All Catalogs
+3. Configuration > Add a New Catalog
+1. `Name = Provisioning`
+1. `Description = Provisioning Services`
+1. Click Save
+4. Catalog Items > All Catalog Items > Provisioning
+5. Configuration > Add a New Catalog Item
+6. `Catalog Item Type = VMWare`
+7. Tab: Basic Information
```

Comment:
Provide the following information in the *Basic Information* tab.

Reply:
Where:
Name: Name of the catalog. Example name can be `VM with Size`
Description: The description of the catalog. 
Display in Catalog: Brief Description. Example input

Basically for any dialogue box/tab where user needs to input multiple values or select multiple values. Please use the following notation and same notation should be used in the entire file for consistency.

`FieldName`: Description of the field. Example input here if needed.
`FieldName1`: Description of the field. Example input here if needed.